### PR TITLE
Plane: move servo wiggle to mode auto

### PR DESCRIPTION
**Pull request by IamPete1**
More plane tidy ups. This removes the early return in the `set_servos` function. New auto test added to look for the wiggle, it passes before and after the plane changes.

### CODE REVIEW

**Comment by IamPete1** [comment]
This change also means that the wiggles will propagate onto elevons and other derived control surfaces.

It still does not work for twin engine planes.

File: ArduPlane/commands_logic.cpp [hunk 1]
```diff
@@ -23,9 +23,6 @@ bool Plane::start_command(const AP_Mission::Mission_Command& cmd)
         // except in a takeoff
         auto_state.takeoff_complete = true;
 
-        // start non-idle
-        auto_state.idle_mode = false;
```
Comment (tridge) [inline ArduPlane/commands_logic.cpp:27]: can remove idle_mode variable now?

**Review by tridge — CHANGES_REQUESTED** [review]
looks good, thanks! 
fine to merge when idle_mode removed

File: ArduPlane/commands_logic.cpp (hunk 1 above)
Comment (IamPete1) [reply ArduPlane/commands_logic.cpp:27]: yep, done.

**Review by tridge — APPROVED** [review]

**Pull request merged**